cpp/oneapi/dal/algo/triangle_counting/backend/cpu/vertex_ranking_default_kernel_scalar.hpp

```diff
@@ -20,6 +20,8 @@
 
 namespace oneapi::dal::preview::triangle_counting::backend {
 
+using namespace oneapi::dal::preview::detail;
+
 template <typename Index>
 ONEDAL_FORCEINLINE std::int64_t intersection(const Index* neigh_u,
```

Comment:
in this case we use neigh_v and below - neighbors - can be aligned in either way

Reply:
This was not aligned. The only aligned thing as part of demonstration is triangle_counting_local_